sdk-lib/src/test/java/cash/z/ecc/android/sdk/ext/ConversionsTest.kt

```diff
@@ -9,7 +10,7 @@ internal class ConversionsTest {
 
     @Test
     fun `default right padding is 6`() {
-        assertEquals(1.13.toZec(6), 113000000L.convertZatoshiToZec())
+        assertEquals(1.13.toZec(6), Zatoshi(113000000L).convertZatoshiToZec())
```

Comment:
Is this from another PR?

Reply:
Yes I found a few stragglers from the prior PR for Zatoshi.  In this PR, I've tried to isolate them to a separate commit.

Reply:
ok! np!. just saying!